views/userpageexternal.hbs

```diff
@@ -0,0 +1,200 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>DeadLock - User Page</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <!-- JavaScripts -->
+    <script src="https://www.w3schools.com/lib/w3.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+    <script src="https://momentjs.com/downloads/moment.js"></script>
+
+    <!-- CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans">
+
+    <link rel="stylesheet" href="/stylesheets/userpagestyle.css">
+    <link rel="stylesheet" href="/stylesheets/standardstyle.css">
+  </head>
+  <body>
+    <nav class="navbar navbar-inverse navbar-fixed-top navbar-custom navbar-custom" role="navigation">
+    <div class="container">
+      <!-- Deadlock logo -->
+      <div class="navbar-header">
+        <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#myNavbar">
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>                        
+        </button>
+        <a class="navbar-brand" href="/"><img src="/images/" width="100%" height="100%" class="d-inline-block align-top" alt="Deadlock"></a>
+        <!-- <a href="/"><img src="images/lock.png" style="margin:5px; width: 20%;height: 20%;" ></a>   --> 
+      </div>
+
+      <div class="collapse navbar-collapse" id="myNavbar">
```

Comment:
Why are you making a whole new navbar? Would it have been possible to instead modify `views/header.hbs`?